guacamole-common-js/src/main/webapp/modules/Mouse.js

```diff
@@ -299,7 +305,7 @@ Guacamole.Mouse = function Mouse(element) {
      * hotspot coordinates. This affects styling of the element backing this
      * Guacamole.Mouse only, and may fail depending on browser support for
      * setting the mouse cursor.
-     * 
```

Comment:
Please remove these whitspace-only changes. I know that removing trailing whitespace improves the formatting, but it clutters up the git history. 

Formatting changes like this should only be included if the lines in question are actually being meaningfully modified.